### DISCUSSION TITLE
CrashPlan-Pro install using Docker Compose

### DISCUSSION THREAD

**Discussion by jacko306** [jlesage/docker-crashplan-pro#451] [Q&A] [answered]
I've been using Crashplan-pro for a few years and installed it on Docker using a script via ssh into my Synology.  I've now moved to container manager and would like to use docker composer but when running the script and logging into the web interface it asked me for a server address under my username? Is this a docker composer issue or because I've installed it from new?
Can I also ask how you include USECRASHPLAN_SRV_MAX_MEMR_ID, GROUP_ID and CRASHPLAN_SRV_MAX_MEM in the composer file?

**Comment by jlesage** [comment, marked as answer]
Since you are probably using the Small Business plan, you should add the `CRASHPLAN_SERVER_ADDRESS` environment variable and set its value to `SMB`.  See https://github.com/jlesage/docker-crashplan-pro#environment-variables for more details.

Here is an example of a section to add to your compose file"
```
  environment:
    - USER_ID=1000
    - GROUP_ID=1000
    - CRASHPLAN_SRV_MAX_MEM=2G
    - CRASHPLAN_SERVER_ADDRESS=SMB
```
https://docs.docker.com/compose/environment-variables/set-environment-variables/

**Comment by jacko306** [comment]
Thanks for responding so quickly, that worked.
